lib/SPIRV/libSPIRV/SPIRVOpCode.h

```diff
@@ -71,7 +71,9 @@ inline bool isAtomicOpCode(Op OpCode) {
 }
 inline bool isBinaryOpCode(Op OpCode) {
   return ((unsigned)OpCode >= OpIAdd && (unsigned)OpCode <= OpFMod) ||
-         OpCode == OpDot || OpCode == OpIAddCarry || OpCode == OpISubBorrow;
+         OpCode == OpDot ||
+         ((unsigned)OpCode >= OpIAddCarry &&
```

Comment:
I think it is clearer if every OpCode that is included is listed.  Using ranges (i.e. OpCode>=X && OpCode<=Y) is harder to understand.

Reply:
Applied